packages/shared/tailwind.config.js

```diff
@@ -1,6 +1,11 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 module.exports = {
+    presets: [require('@frontify/fondue/tailwind')],
+    content: ['src/**/*.{ts,tsx}'],
```

Comment:
```suggestion
    presets: [require('@frontify/fondue/tailwind'), require('../shared/tailwind.config')],
    content: ['src/**/*.{ts,tsx}', '../shared/src/**/*.{ts,tsx}'],
```

Reply:
This is already inside the shared folder

Reply:
omg goldfish brain. :D